package.json

```diff
@@ -34,7 +34,7 @@
         "pg": "^8.4.2",
         "piscina": "^2.1.0",
         "posthog-js-lite": "^0.0.5",
-        "posthog-plugins": "^0.2.3",
+        "posthog-plugins": "^0.2.4",
```

Comment:
This should probably be pinned to a specific version if this is actually important.

Reply:
pinned!